src/com/aca/homework/week4/snake/Snake.java

```diff
@@ -0,0 +1,146 @@
+package com.aca.homework.week4.snake;
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class Snake {
+    private char[][] board;
+    private int[][] snake;
+    private int snakeSize = 1;
+    private int appleRow;
+    private int appleCol;
+    private boolean running = true; // if the player loses, the variable becomes false
+
+    public Snake(char[][] board, int headRow, int headCol, int appleRow, int appleCol) {
+        this.board = board;
+        this.snake = new int[board.length*board[0].length][2];
+        this.snake[snakeSize-1][0] = headRow;
+        this.snake[snakeSize-1][1] = headCol;
+        this.board[headRow][headCol] = '*';
+        this.appleRow = appleRow;
+        this.appleCol = appleCol;
+        this.board[appleRow][appleCol] = '=';
+    }
+    public String getMove(){
+        Scanner scanner = new Scanner(System.in);
+        String move = scanner.nextLine();
+        return move;
+    }
+    public void move(){
```

Comment:
you have a lot of if conditions in your method. try to split the method to smaller ones